src/pages/Blog/styles.ts

```diff
@@ -0,0 +1,141 @@
+import { makeStyles, Theme } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme: Theme) => ({
+  root: {
+    position: "relative",
+    backgroundImage: `url(${"/assets/images/faq_background.png"})`,
```

Comment:
`faq_background`? Is this same background used in faq page?

Reply:
yes, the same background has been used.